addons/l10n_ch/views/account_report.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <!-- QWeb Reports -->
+        <record id="l10n_ch_qr_server_action" model="ir.actions.server">
+            <field name="name">Print QR Invoices</field>
+            <field name="model_id" ref="account.model_account_move"/>
+            <field name="binding_model_id" ref="account.model_account_move"/>
+            <field name="binding_view_types">list</field>
+            <field name="state">code</field>
+            <field name="code">
+if records:
+    action = records.l10n_ch_action_print_qr()
+            </field>
+        </record>
+    </data>
```

Comment:
I'm not convinced by the file name... In other modules, server actions with binding views are put inside the `..._views` file